testing/github-11072/config.toml

```diff
@@ -0,0 +1,19 @@
+data_dir = "/tmp/vector/github-11072"
```

Comment:
Not sure if you meant to commit your `testing/` directory 😄

Reply:
This is by design, actually!

The goal of `testing/` is to version control the testing plan for issues/PRs so that they show up in code review so others can review the test the same as the fix, but also so that it can live on for providing historical context.